CIP-0094/README.md

```diff
@@ -0,0 +1,281 @@
+```
+
+Some remarks:
+
+1. The field `2` (`question_hash`) is a blake2b-256 hash digest, whose preimage is the entire serialised question metadata payload (with the `94` top-level label).
+1. The field `3` represents the 0-based index of the chosen answer from the available choices (from field `1` of the target poll).
+
+For example:
+
+<table>
+<thead>
+  <th>CBOR diagnostic</td>
+  <th>Base16</th>
+</thead>
+<tbody>
+ <tr>
+  <td>
+<pre>
+{
+  94: {
+    2: h'29093fd43fc30ba31e306af06ce8537390e1668ae7496fe53d53684683c3762c',
+    3: 0
+  }
+}
+</pre>
+  </td>
+  <td>
+   <a target="_blank" href="https://cbor.me/?bytes=A1185EA202582029093FD43FC30BA31E306AF06CE8537390E1668AE7496FE53D53684683C3762C0300">
+<pre>
+A1185EA202582029093FD43FC30BA31E306AF06CE
+8537390E1668AE7496FE53D53684683C3762C0300
+</pre>
+   </a>
+  </td>
+ </tr>
+</tbody>
+</table>
+
+The transaction carrying the answer metadata must then **be signed using a stake pool operator cold key**. Because cold key are not payment keys, this requires specifying an extra required signer on the transaction (transaction's field number 14 as per [Babbage's CDDL](https://github.com/input-output-hk/cardano-ledger/blob/cffa75fdbd800cda60997791e51bf02f2af0c42b/eras/babbage/test-suite/cddl-files/babbage.cddl#L66)). The signature shall be produced in an air-gapped environment only.
+
+> **Warning**
+>
+> Only the first answer to a poll for each credential shall be considered. If multiple answers are found, only the first answer submitted (transaction & block ordering tallying) shall be considered.
+
+### Procedure and Duration
+
+A poll starts when a valid transaction with a question is posted on-chain. Ballots can be submitted until the end of the following epoch, so there is always at least one whole epoch to answer the poll.
+
+After the epoch in which the Stake Pool Operators have cast their votes, there follows a period of one or more epochs in which the ADA Delegates may respond: If they disagree with the choice of their current Stake pool, they can delegate to another pool. This changes the stake weight and thus influences the result.
+
+![Alt text](CIP-0094_procedure-duration.png "Epoch poll phases")
+
+Indirectly, this results in the possibility of participation for all ada holders. Only after this re-delegation epoch(s) is completed, is the stake snaphot of the pools made to calculate the outcome. 
+
+### Outcome
+
+The outcome of a poll will depend on its level of participation (in **terms of stake**). It is essential to understand that we explicitly call this a _poll_ / _survey_ and not a _vote_ to dispel any possible confusion. So it is akin to `1 Lovelace = 1 Voice` although we may chose to interpret data using different equations (e.g. giving more weight to pledged stake). How the data is interpret is deemed out of the scope of this proposal which aims mainly at producing the data-points. Further conversations and debates will be needed regarding interpretation of the data-points.
```

Comment:
> we explicitly call this a _poll_ / _survey_ and not a _vote_

Except in the `Procedure and Duration` section where its referenced as a vote :)

Reply:
fixed this